packages/server/controllers/auth.js

```diff
@@ -189,6 +190,84 @@ async function authenticate(request, response, next) {
   }
 }
 
+async function isUserEmail(email) {
+  const result = await prisma.users.findUnique({
+    where: {
+      email: email,
+    },
+  });
+
+  if (result === null) return false;
+
+  return true;
+}
+
+async function isGoogleAccount(userId) {
+  const result = await prisma.users.findUnique({
+    where: {
+      userId: userId,
+    },
+  });
+
+  if (result.password == null) return true;
+
+  return false;
+}
+
+async function sendEmail(address, subject, body) {
+  const transporter = nodemailer.createTransport({
+    host: "mail.privateemail.com",
+    port: 465,
+    secure: true,
+    auth: {
+      user: "icebreak@cppicebreak.com",
+      pass: process.env.MAILBOX_PASSWORD,
+    },
+  });
+
+  const info = await transporter.sendMail({
+    from: "icebreak@cppicebreak.com",
+    to: address,
+    subject: subject,
+    text: body,
+  });
+
+  return info;
+}
+
+async function sendPasswordResetEmail(address, passwordResetToken) {
+  const subject = "Icebreak: Password Reset Request";
+  const link = `http://localhost:5050/reset-password?token=${passwordResetToken}`;
+  const body = `Please click the following link to reset your password: ${link}`;
+
+  sendEmail(address, subject, body);
+}
```

Comment:
I believe this needs to be
```js
return sendEmail(address, subject, body);
```
since you're assigning the result of this function to a variable in the forgot passsword API route but this function doesn't return anything.